.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -105,12 +105,16 @@ body:
         - "3.11"
         - "3.10"
 
-  - type: markdown
+  - type: textarea
+    id: screenshot
     attributes:
-      value: |
-        Screenshot error - If applicable, add screenshots to help explain your problem.
+      label: Screenshot
+      description: "If applicable, add screenshots to help explain your problem."
+      placeholder: "Paste your screenshot here."
 
-  - type: markdown
+  - type: textarea
```

Comment:
why textarea?

Reply:
I made this change so the user has space to drop in the screenshot.

<img width="413" alt="image" src="https://github.com/user-attachments/assets/c228f592-163c-4235-9195-7657a7b74a87">

Reply:
With markdown they could upload files and images directly, I think.
![CleanShot 2024-07-24 at 14 25 04@2x](https://github.com/user-attachments/assets/9e05ef4a-f130-4b4f-a0f2-02495f562856)

Reply:
I don't know if that's possible with textarea.

Reply:
This isn't for uploading an image. The user will either drag and drop or paste a screenshot directly with CTRL + V. From what I've tested, YAML doesn't support image uploads.

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

It seems textarea works with image uploads.